11.0.Dockerfile

```
@@ -195,11 +182,31 @@ ONBUILD ENV ADMIN_PASSWORD="$ADMIN_PASSWORD" \
             WITHOUT_DEMO="$WITHOUT_DEMO"
 ONBUILD ARG LOCAL_CUSTOM_DIR=./custom
 ONBUILD COPY $LOCAL_CUSTOM_DIR /opt/odoo/custom
+
+# Enable setting custom uids for odoo user during build of scaffolds
+ONBUILD ARG UID=1000
+ONBUILD ARG GID=1000
+
+# Enable Odoo user and filestore
+ONBUILD RUN groupadd -g $GID odoo \
+    && useradd -md /home/odoo -s /bin/false -u $UID -g $GID odoo \
+    && mkdir -p /var/lib/odoo \
+    && chown -R odoo:odoo /var/lib/odoo \
```

I'm seeing that this can be a single command:

```suggestion
    && chown -R odoo:odoo /var/lib/odoo /qa/artifacts \
```

---

refactored